CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ This release has an [MSRV] of 1.82.
 
 - `ImageQuality` and `ImageFormat` now optionally impl `bytemuck` traits ([#104][] by [@waywardmonkeys][])
 
+### Changed
+
+- Update to `color` 0.3.0. ([#103][] by [@waywardmonkeys][])
```

Comment:
Should we mark this as breaking?

Reply:
(It could be worth also thinking about whether stuff like this should link to Color directly)